contents/tutorials/how-to-run-successful-group-experiments.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to run successful group-level A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: // what to put here?
+featuredImage: //TODO
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one person uses your product will affect how others interact with it too. For example, a change in the Uber App for Drivers will likely affect the experience of riders too. In these scenarios, group-level experiments allow us to measure the impact of changes beyond an individual and across a group of users.
+
+// TODO: Add image showing difference between group-level and user-level experiment
+
+## When to run group-level experiments
+
+When the change in how an individual uses your product will greatly affect the behavior of other users, or when you want to measure the impact of a change on an entire group of users. This is usually seen in the following types of products:
+
+1. B2B SaaS apps
+
+   - For example, suppose Asana wants to test a new AI feature that automatically assigns tasks based on their urgency and importance. By conducting a group-level experiment, they can measure the impact on project completion rates across the entire company. 
+
+2. Products with network effects.
```

Comment:
Is this relevant to products with individual network effects? Or just group network effects?

Reply:
Hmm Im not sure I understand if there would be a difference? Can you elaborate on what you mean?

Reply:
I was thinking something, but I think what you have as right. 

For example, Facebook has individual-level network effects. New individuals benefit existing individuals. Group network effects is like new organizations benefiting existing organizations. For example, if a new organization joins Slack that benefits existing organizations who want to Slack Connect with them. 

Again, I think I'm wrong and you're right 😄